clients/adapter/src/main/java/org/eclipse/hono/adapter/client/command/Command.java

```diff
@@ -97,7 +97,10 @@
     String getOriginalDeviceId();
 
     /**
-     * Gets the request identifier of this command.
+     * Gets the request identifier of this command. Can be used to correlate this command
+     * with a corresponding command response.
+     * <p>
+     * May be {@code null} for a one-way command.
```

Comment:
It *will* be `null` for a one-way command, won't it?

Reply:
For the `KafkaBasedCommand`, yes.
But not in the case of a `ProtonBasedCommand` with a `correlation-id` property but no `reply-to` property (i.e. a one-way command with a given correlation-id).